### PR TITLE
📖amp-list: documentation for using the new strip-prefix attribute

### PR DESCRIPTION
**Pull request by samouri**
Partially addresses https://github.com/ampproject/amphtml/issues/25593

cc @caroqliu / @choumx

### CODE REVIEW

File: extensions/amp-list/amp-list.md [hunk 1]
```diff
@@ -294,6 +294,22 @@ The `items` array will be truncated to `max-items` entries if the returned value
 Causes `<amp-list>` to treat the returned result as if it were a single element array. An object response will be wrapped in an array so
 `{items: {...}}` will behave as if it were `{items: [{...}]}`.
 
+##### strip-prefix (optional)
+
+Causes `<amp-list>` to strip a prefix from a given endpoint's JSON before parsing. This can be useful for APIs that include [security prefixes](http://patorjk.com/blog/2013/02/05/crafty-tricks-for-avoiding-xssi/) like `)]}` to help prevent cross site scripting attacks.
```
Comment (choumx) [inline extensions/amp-list/amp-list.md]: "from a given endpoint's JSON" -> "from the fetched JSON"

The former sounds like there's a choice from multiple endpoints.

File: extensions/amp-list/amp-list.md [hunk 2]
```diff
@@ -294,6 +294,22 @@ The `items` array will be truncated to `max-items` entries if the returned value
...
 `{items: {...}}` will behave as if it were `{items: [{...}]}`.
 
+##### strip-prefix (optional)
+
+Causes `<amp-list>` to strip a prefix from the fetched JSON before parsing. This can be useful for APIs that include [security prefixes](http://patorjk.com/blog/2013/02/05/crafty-tricks-for-avoiding-xssi/) like `)]}` to help prevent cross site scripting attacks.
+
+For example, lets say we had an API that returned this response:
+
+```
+)]}{ 'a': 'value' }
```
Comment (caroqliu) [inline extensions/amp-list/amp-list.md]: Let's make this example data shape similar to the JSON that `amp-list` typically expects to receive:
`{ "items": [...] }`

Comment (samouri) [reply extensions/amp-list/amp-list.md]: Done, PTAL

File: extensions/amp-list/amp-list.md [hunk 3]
```diff
@@ -312,7 +328,7 @@ We recommend using `binding="no"` or `binding="refresh"` for faster performance.
 
 If `binding` attribute is not provided, default is `always`.
 
-##### [is-layout-container](optional)
+##### is-layout-container (optional)
```
Comment (caroqliu) [inline extensions/amp-list/amp-list.md]: Looks like this is a bindable attribute, so it should keep the brackets, as well as the newly added space character.

Comment (samouri) [reply extensions/amp-list/amp-list.md]: Thanks for letting me know! I didn't know what bindable attrs get square brackets in our docs. 

In `master` right now, it lives as a broken link in the amp-list documentation. This should fix it up.

Comment (caroqliu) [reply extensions/amp-list/amp-list.md]: Thanks for cleaning things up as you go!

**Pull request merged**